scss/tokens/_raw.scss

```diff
@@ -20,6 +20,7 @@ $ouds-border-radius-300:  $ouds-border-base * 3 !default;
 // $ouds-border-radius-500:  $ouds-border-base * 5 !default;
 // $ouds-border-radius-600:  $ouds-border-base * 6 !default;
 // $ouds-border-radius-800:  $ouds-border-base * 8 !default;
+$ouds-border-radius-9999: 2000 !default;
```

Comment:
yep we should have px here, it is missing in Tokenator output because I excluded border-radius token since they were all calculated from border base. I must fix that, we need the `px` here ! :)

Reply:
Fixed with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2751/commits/e1fe493322edb95bac26b3d1120ac1beb7306ade